src/actions/layers/VideoLayer.ts

```diff
@@ -0,0 +1,99 @@
+/**
+ * Video Layer used in Overlays/Underlays
+ * @memberOf Actions
+ * @namespace Layers
+ */
+
+
+import {ILayerAction} from "./ILayerAction";
+import Action from "../Action";
+// TODO - BundleSize Warning - we include all the Sources code within Layers.
+import Source, {ImageSource, TextSource} from "../../params/sources/Sources";
+import {Position} from "../../params/position/Position";
+import Param from "../../parameters/Param";
+import {BlendMode} from "../../params/blendMode/BlendMode";
+import {ISource} from "../../params/sources/ISource";
+
+
+class VideoLayer extends Action implements ILayerAction {
```

Comment:
Can we extend ImageLayer or Layer maybe?
It might save us all the code duplication

Reply:
I removed this whole file as it's not needed, just like we don't have an ImageLayer.ts